### ISSUE TITLE
can't open flf file 

### ISSUE THREAD

**Issue by andriy-byte** [srilakshmikanthanp/libfiglet#5]

**Comment by srilakshmikanthanp** [comment]
Thanks for raising the issue,  Did you give the correct path to the font? If yes, can you provide the sample code and the stack trace?

**Comment by andriy-byte** [comment]
> Thanks for raising the issue, Did you give the correct path to the font? If yes, can you provide the sample code and the stack trace?

i resolved the file opening problem, they open's , and i got another problem, some flf files with correct token cant be rendered, maybe problem is in numbers after token, i tried to render stardart.flf, the rendering was incorrect, the letters were wrapped with the @ symbol.

Thanks for your work, there is no c++ figlet renderer besides yours, but it has little bit problems.

**Comment by srilakshmikanthanp** [comment]
Can you provide the sample code that renders wrong with output?

**Comment by andriy-byte** [comment]
> Can you provide the sample code that renders wrong with output?

i use font which are in assets/fonts, in this case i use Big.flf 
```
#include <iostream>
#include "libfiglet.hpp"
using namespace srilakshmikanthanp::libfiglet;

int main()
{
    const figlet figlet(flf_font::make_shared("../Big.flf"), smushed::make_shared());
    std::cout << figlet("GOLDEN LEAF COIN");
}
```
and recived runtime error
```
terminate called after throwing an instance of 'std::runtime_error'
  what():  Invalid flf header
```
same problem with Cyberlarge.flf font , other fonts  I didn't test, I think that the problem doesn't apply to all fonts because Standard.flf works

**Comment by srilakshmikanthanp** [comment]
Thanks For raising the issue Fixed with 6490fb1, Hope the [v1.2.1](https://github.com/srilakshmikanthanp/libfiglet/releases/tag/v1.2.1) Fixes it. If you still facing any issues please reopen the issue.
